examples/custom-colors/index.html

```diff
@@ -63,9 +63,9 @@ <h2>Custom colors for User Task</h2>
     </section>
 
 <!-- load global settings -->
-<script src="../../demo/0.5.1-alpha/static/js/configureMxGraphGlobals.js"></script>
+<script src="../../demo/0.6.0-alpha/static/js/configureMxGraphGlobals.js"></script>
```

Comment:
```suggestion
<script src="../../demo/0.6.0/static/js/configureMxGraphGlobals.js"></script>
```

Reply:
good catches, I have checked that I didn't miss 0.5.1 but forgot to check alpha.
We should probably avoid using alpha next time to avoid such miss.

I have managed all alpha removal with commit 11605b3